docs/source/migrations/modes.rst

```diff
@@ -0,0 +1,55 @@
+======================
+Snuba Migration Modes
+======================
+
+This doc outlines a basic way to try out distributed and local migrations.
+Note that this is experimental, and should be used only for development
+purposes at the moment.
+
+The main "switch" between the two modes for running data migrations (local and
+distributed) lives in ``sentry/conf/server.py``.
+The controlling envrionment variable is ``SENTRY_DISTRIBUTED_CLICKHOUSE_TABLES``,
+and its value must be set in order to use a specific mode.
+
+Once this boolean variable is set, one of two Clickhouse Docker volumes will be
+used for data storage, depending on the mode (distributed or local). Whenever a user
+wants to switch between the two modes, they must "turn off" the running Clickhouse
+container, alter the environment variable mentioned above, and then "turn on" the
+same container to be in the new mode.
+
+More information on migrations in general can be found `here <https://github.com/getsentry/snuba/blob/master/MIGRATIONS.md>`_.
+
+Enabling Local Mode
+=====================
+
+In your local ``server.py``, set ``SENTRY_DISTRIBUTED_CLICKHOUSE_TABLES``
+to False. Start up the corresponding Clickhouse container (``sentry devservices up clickhouse``).
+
+Ensuring that your local configuration is prepared for migrations (reverting any existing
+migrations, or dropping certain databases/tables), run migrations as expected
+(``snuba migrations migrate --force``).
```

Comment:
I don't think this is necessary. People should not manually revert migrations or drop tables/databases. They don't even need to run the migrate script manually since the devserver just does that.

Reply:
I tested it and you're right; removed it.